pwndbg/gdblib/disasm/instruction.py

```diff
@@ -333,6 +337,12 @@ def __repr__(self) -> str:
         Can change PC: {self.can_change_instruction_pointer}
         Syscall: {self.syscall if self.syscall is not None else ""} {self.syscall_name if self.syscall_name is not None else "N/A"}"""
 
+        # Hacky, but this is just for debugging
+        if hasattr(self.cs_insn, "cc"):
+            info += f"\n\tARM condition code: {self.cs_insn.cc}"
```

Comment:
Shall we remove this?

Reply:
Probably not since its repr anyway?

Reply:
Although it's hacky, I think this is the best that we can currently do (I really do not want to subclass PwndbgInstruction just for a debug statement in a particular architecture). This print is only shown using the `dev-dump-instruction` command which is invaluable for debugging